Userland/Utilities/winlist.cpp

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2022, demostanis worlds <demostanis@protonmail.com>
+ * Copyright (c) 2022, the SerenityOS developers.
```

Comment:
Why two copyright holders?

Reply:
I just wanted to have my name somewhere and saw some other files do it. Guess I'll keep "the SerenityOS developers"

Reply:
I didn't mean to discourage you to put your name on the changes, just adding two copyright headers refering to the same author for this change is weird.

If you use `demostanis worlds <demostanis@protonmail.com>` then its probably better to do so for all the files